packages/engine-test-utils/src/__tests__/engine-server/store/noteStore.spec.ts

```diff
@@ -0,0 +1,393 @@
+import { ERROR_STATUS, NotePropsMeta } from "@dendronhq/common-all";
+import { NoteTestUtilsV4 } from "@dendronhq/common-test-utils";
+import {
+  DendronEngineClient,
+  NodeJSFileStore,
+  NoteMetadataStore,
+  NoteStore,
+} from "@dendronhq/engine-server";
+import _ from "lodash";
+import { runEngineTestV5 } from "../../../engine";
+import { ENGINE_HOOKS } from "../../../presets";
+
+describe("NoteStore", () => {
```

Comment:
can you write the test in the test style? [[Writing Tests|dendron://dendron.docs/dev.process.qa.test#writing-tests]]

GIVEN "NoteStore", WHEN running note.find, THEN ...

Reply:
same comment for other tests